### PR TITLE
KAA-1554: Update sample applications build instructions in C/C++ SDKs documentation pages

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @Slash32 @vadimol

### CODE REVIEW

**Review by Slash32 — APPROVED** [review]

**Comment by vadimol** [comment]
Please, resolve merge conflicts.

**Review by Slash32 — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md [hunk 1]
```diff
@@ -150,14 +150,10 @@ Connect CC3200 LaunchPad to your PC through a micro-USB connector and execute th
...
-
-You will be asked for desired platform. Type `cc32xx` into the console.
-
-```
-Please enter a target (default is x86-64):
-cc32xx
+tar -zxf libs/kaa/kaa-* -C libs/kaa
+mkdir -p build
+cd build
+cmake -DKAA_PLATFORM=cc32xx -DCMAKE_TOOLCHAIN_FILE=../libs/kaa/toolchains/cc32xx.cmake ..
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:156]: Is path correct? See https://github.com/kaaproject/kaa/pull/1249

Comment (vadimol) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:156]: It is correct here. We referred to the sample application above.

**Review by vadimol — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md (hunk 1 above)
Comment (rasendubi) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md:156]: ok, I see. LGTM then

**Review by rasendubi — APPROVED** [review]

**Pull request merged**